Code/Tools/FBuild/FBuildCore/Cache/CachePlugin.cpp

```diff
@@ -64,20 +79,18 @@
 
 // GetFunction
 //------------------------------------------------------------------------------
-void * CachePlugin::GetFunction( const char * friendlyName, const char * mangledName ) const
+void * CachePlugin::GetFunction( const char * friendlyName, const char* mangledName) const
 {
     #if defined( __WINDOWS__ )
         ASSERT( m_DLL );
-        void * func = ::GetProcAddress( (HMODULE)m_DLL, mangledName );
+        void * func = ::GetProcAddress( (HMODULE)m_DLL, name );
```

Comment:
Build is broken on dev because of this :P
````
Code\Tools\FBuild\FBuildCore\Cache\CachePlugin.cpp(86) : error C2065: 'name' : undeclared identifier
````